__docs__/components/all-variants.tsx

```diff
@@ -141,7 +141,6 @@ export function AllVariants(props: Props) {
                                         >
                                             {children({
                                                 props: {
-                                                    "aria-label": ariaLabel,
```

Comment:
Removing including `aria-label` in the props since it is not always valid to have an `aria-label` depending on the element. This addresses the a11y warning: `Ensure ARIA attributes are not prohibited for an element's role`.

We can explicitly set the aria-label for components in specific stories using the `name` that is also provided to the `children` render function

Reply:
Good call! I updated the aria-label for OptionItemVariants!

https://github.com/Khan/wonder-blocks/blob/37568f0c20014958ee84ffd12680666a9900001e/__docs__/wonder-blocks-dropdown/option-item-variants.stories.tsx#L55